Core/include/Acts/Vertexing/HelicalTrackLinearizer.hpp

```diff
@@ -94,13 +94,15 @@ class HelicalTrackLinearizer {
       : m_cfg(config), m_logger{std::move(_logger)} {}
 
   /// @brief Function that linearizes BoundTrackParameters at
-  /// given linearization point
+  /// the PCA to a given Perigee surface
   ///
   /// @param params Parameters to linearize
-  /// @param linPoint Linearization point
-  /// @param gctx The geometry context
-  /// @param mctx The magnetic field context
-  /// @param state The state object
+  /// @param linPoint Point which defines the Perigee.
+  /// Transverse plane of Perigee is parallel to the global 
```

Comment:
Maybe this could be a `@note`?

Reply:
It belongs to the parameter description! Should I make it note nonetheless?

Reply:
You can refer to the parameter like `@p linPoint`, maybe that's good enough to keep the connection?

Reply:
Ok, sounds good!